src/Concordium/Client/Commands.hs

```diff
@@ -926,6 +1016,7 @@ accountDecryptCmd =
                 <$> transactionOptsParser
                 <*> option (maybeReader amountFromString) (long "amount" <> metavar "CCD-AMOUNT" <> help "The amount to transfer to public balance.")
                 <*> optional (option auto (long "index" <> metavar "INDEX" <> help "Optionally specify the index up to which shielded amounts should be combined."))
+                <*> optional (strOption (long "outFile" <> metavar "FILE" <> help "An optional file name and path to ouptput the signed/partially-signed transaction to instead of submitting the transaction on-chain."))
```

Comment:
I think this is not needed since it is already in `transactionOptsParserBuilder` used in `transactionOptsParser` :)